docs/src/main/asciidoc/logging.adoc

```diff
@@ -9,10 +9,95 @@ include::./attributes.adoc[]
 
 This guide explains logging and how to configure it.
 
+Internally, Quarkus uses JBoss Log Manager and the JBoss Logging facade. +
+You can use the JBoss Logging facade inside your code as it's already provided,
+or any of the supported Logging API listed in the next chapter as Quarkus will send them to JBoss Log Manager.
+
+All the logging configuration will then be done inside your `application.properties`.
+
+== Supported Logging APIs
+
+Applications and components may use any of the following APIs for logging, and the logs will be merged:
+
+* JDK `java.util.logging` (also called JUL)
+* https://github.com/jboss-logging/jboss-logging[JBoss Logging]
+* https://www.slf4j.org/[SLF4J]
+* https://commons.apache.org/proper/commons-logging/[Apache Commons Logging]
+
+Internally Quarkus uses JBoss Logging, you can also use it inside your application so that no other dependencies should be added for your logs.
```

Comment:
```suggestion
Internally Quarkus uses JBoss Logging; you can also use it inside your application so that no other dependencies should be added for your logs.
```